lib/ChGovUk/Controllers/Search/CompanyNameAvailability.pm

```diff
@@ -133,13 +133,11 @@ sub get_basket() {
                 my ($api, $tx) = @_;
                 debug "Failure returned by GET basket endpoint; not displaying basket link.", [HOMEPAGE];
                 $self->stash_basket_link(0, undef);
```

Comment:
Will you want to log out error messages in this module too?

Reply:
Sorry, missed that one. Should be done now.